NOTICE

```diff
@@ -78,6 +79,7 @@ the following terms:
   Hauke Mehrtens
   Hiltjo Posthuma
   Isaac Dunham
+  Jay Phelps
```

Comment:
These are the contributors to [musl libc](http://git.musl-libc.org/cgit/musl/tree/COPYRIGHT). Is this correct?

Reply:
Whoops! I am terrible at reading. I have not contributed to musl, will remove thanks!